index.html

```diff
@@ -0,0 +1,154 @@
+              <img src="image/forum.png" alt="">
+              <h5>Fashion</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/workshop.png" alt="">
+              <h5>On Screen</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/ccignite.png" alt="">
+              <h5>On Screen</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+          </div>
+          <a href="" class="see-whole-program">SEE THE WHOLE PROGRM</a>
+          <Button type="button">Book Now for Your Program</Button>
+        </div>
+            <section class="support-team">
+              <h2 class="support-section-titles">Support Teams</h2>
+              <hr class="section-title-separators">
+              <ul class="support-list-items" id="support-list">
+              </ul>
+              <button type="button" class="more">More <i class="fa fa-angle-down"></i></button>
+            </section>
+          </div>
+      <section class="partners-list container">
+        <h2 class="section-titles">Partners</h2>
+        <hr class="section-title-separators">
+        <ul id="partners-list-items" class="partners-list-items">
+          <li class="partners-items">
+            <a target="blanc" href="#">
+              <img src="image/afri-herbal.jpg" alt="Afri herbale Logo" />
+            </a>
+          </li>
+          <li class="partners-items">
+            <a target="blanc" href="#">
+              <img src="image/awash.png" alt="Awash bank logo" />
+            </a>
+          </li>
+          <li class="partners-items">
+            <a target="blanc" href="#">
+              <img src="image/sol-image.png" alt="Sol image logo" />
+            </a>
+          </li>
+          <li class="partners-items">
+            <a target="blanc" href="#">
+              <img src="image/ethi-herbal.jpg" alt="Ethi herbal logo" />
+            </a>
+          </li>
+          <li class="partners-items">
+            <a target="blanc" href="#">
+              <img src="image/barber-chair.jpg" alt="barber wood product logo" />
+            </a>
+          </li>
+        </ul>
+      </section>
+      <footer class="footer">
+        <div class="full-log">
+          <a href="index.html" class="logo-2">
+            <img src="image/logo.png" alt="Faya women beauty salon logo">
+          </a>
+        </div>
+        <div class="footer-copy-right">
+          <p>Faya women beauty salon plc.<br> Address: Addis Ababa,Ethiopia.<br>
+            Office: Cabey Building 3rd floor, Room No. 304.</p>
+        </div>
+      </footer>
```

Comment:
- [ ] Please try to center the footers design to meet the [required design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)